environment/src/main/resources/schema/app/20221123111223_CB-19565_add_skip_org_policy_decisions_column.sql

```diff
@@ -0,0 +1,10 @@
+-- // CB-19565 Create credential property to be able to skip organizational rule validation
+-- Migration SQL that makes the change goes here.
+ALTER TABLE credential ADD COLUMN IF NOT EXISTS skiporgpolicydecisions BOOLEAN DEFAULT FALSE;
+UPDATE credential SET skiporgpolicydecisions = FALSE WHERE skiporgpolicydecisions IS NULL;
```

Comment:
Do you have a rough estimate of how many records are affected?

Reply:
I have no estimate, I assume it is less than the number of the envs.